chapters/chapter_2.md

```diff
@@ -157,7 +157,7 @@ This should land you in a text editor, usually vi(m), where you add commit title
 #
 ```
 
-This is giving the output you saw with `git status` one more time, so that you can be sure that you are committing all the files you want. On the first line, give a commit title. Leave a blank line, and from the third line, you can write the description.
+The is giving the output you saw with `git status` one more time, so that you can be sure that you are committing all the files you want. On the first line, give a commit title. Leave a blank line, and from the third line, you can write the description.
```

Comment:
Probably not needed

Reply:
It seems like the error was many consecutive sentences starting with "This". Should've corrected